### PR TITLE
Test Web applications

### PR DESCRIPTION
**Pull request by slinkydeveloper**

### CODE REVIEW

File: src/test/java/io/vertx/howtos/web/test/WebApplicationTest.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  }
+  // end::tearDown[]
+
+  // tag::testGetPetOk[]
+  @Test
+  public void testGetPetOk(WebClient client, VertxTestContext testContext) {
+    testRequest(
+      client    // Create the test request using WebClient APIs
+        .get("/pet/1")
+    )
```
Comment (jponge) [inline src/test/java/io/vertx/howtos/web/test/WebApplicationTest.java]: You can restyle to something like:

```java
client
  .get("...")
  .expect(//...
  )
  .send(testContext)
```

File: src/test/java/io/vertx/howtos/web/test/WebApplicationTest.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  }
+  // end::testGetPetOk[]
+
+  // tag::testGetPetErrors[]
+  @Test
+  public void testGetPetBadRequest(WebClient client, VertxTestContext testContext) {
+    testRequest(
+      client
+        .get("/pet/bad")
+    )
```
Comment (jponge) [inline src/test/java/io/vertx/howtos/web/test/WebApplicationTest.java]: Same over here

**Comment by jponge** [comment]
Next steps for you @slinkydeveloper 

- [ ] Make edits
- [ ] Merge the PR
- [ ] Setup CI with your Travis token

**Comment by tsegismont** [comment]
👏

**Comment by slinkydeveloper** [comment]
@jponge i can't manage to configure travis to build this repo, can you do it for me?

**Comment by jponge** [comment]
The repos has been Travis-enabled, you can now add your key @slinkydeveloper

**Comment by jponge** [comment]
👍

**Pull request merged**